### ISSUE TITLE
Unable to locate publishable resources.

### ISSUE THREAD

**Issue by paishin** [marcreichel/igdb-laravel#35]
I'm on Laravel 6.20.27 running igdb-laravel 2.1.1

Plugin installs fine but when I try to publish the configuration file I get an error saying "Unable to locate publishable resources."
I have tried everything that comes to mind but I just cant get this to work. Could you please assist me?

```
$ php artisan vendor:publish --provider="MarcReichel\IGDB-Laravel\IGDBLaravelServiceProvider"
Unable to locate publishable resources.
```

**Comment by marcreichel** [comment]
Well, you have a typo in your command. It has to be `IGDBLaravel`, not `IGDB-Laravel`:

```
php artisan vendor:publish --provider="MarcReichel\IGDBLaravel\IGDBLaravelServiceProvider"
```

**Comment by paishin** [comment]
@marcreichel, thank you so much for the quick response, 

unfortunately the typo was made while writing the question not how I was actually running the command, so I am still facing this issue.
![image](https://user-images.githubusercontent.com/410381/119241435-f9f78400-bb5e-11eb-9923-8c5d0e8dd569.png)

Even more strange is that the package does not appear while running `php artisan vedor:publish` either
![image](https://user-images.githubusercontent.com/410381/119241458-1bf10680-bb5f-11eb-8e40-5b1f4ee4c55d.png)

but it is there when running `php artisan package:discover`
![image](https://user-images.githubusercontent.com/410381/119241509-7f7b3400-bb5f-11eb-9358-fd163975a766.png)

manually creating igdb.php file in config folder has no effect nor does adding the credentials in .env
I have edited the files in vendor folder with my credentials just to test it out and that makes the package work flawlessly.

Please let me know if I could provide you with any more details

**Comment by marcreichel** [comment]
No worries, this sounds like you have cached your configuration with `php artisan config:cache`. Please try clearing your config cache with `php artisan config:clear` and then try again publishing the config. Then the credentials in your `.env` file should work as well because the `.env` is not used while the config is cached.

Feel free to get back to me if this does not do the trick.

**Comment by paishin** [comment]
I had already tried that and it does not solve the issue. I have re-checked just in case, but the problem persists.
![image](https://user-images.githubusercontent.com/410381/119241974-f49c3880-bb62-11eb-8159-b058a2ee3d7e.png)
![image](https://user-images.githubusercontent.com/410381/119241992-15fd2480-bb63-11eb-9031-ee952c26788c.png)

**Comment by paishin** [comment]
Update, although I can still not publish the resource automatically, creating the igdb.php file in config file works now I have cleared the cache. 

If no one else mentioned this issue then it might be something with my project specifically so mark this as complete.

@marcreichel thank you so much for your support

**Comment by marcreichel** [comment]
Okay, happy to help (at least half way through 😅).

I will look into this issue with your provided versions though just to be sure if it's a general problem or not 👍🏼

So I leave it open for now 👍🏼

**Comment by marcreichel** [comment]
@paishin So, just to give you an update: I've created a fresh test project with Laravel v6.20.27 and installed this package with version 2.1.1 and I was able to publish the config file with `php artisan vendor:publish --provider="MarcReichel\IGDBLaravel\IGDBLaravelServiceProvider"`. So it seems to not be a problem with this package you've faced but with your project somehow.
Are you able to publish config files (or views e.g.) from other packages?
